compiler/include/flags.h

```diff
@@ -24,7 +24,8 @@
 
 //
 // HOW TO ADD A FLAG:
-// - add an entry (sorted alphabetically) to the list in flags_list.h
+// - add an entry (sorted alphabetically) to the list in
+//   compiler/dyno/include/chpl/uast/PragmaList.h
```

Comment:
Was this intentionally included in this PR?  I think it should be mentioned in the PR message.